kurobako/src/enginemain.cpp

```diff
@@ -1,31 +1,31 @@
 #include "enginemain.h"
 #include "kurobakoconfig.h"
-
-#include "graphics\hello_triangle.h"
-
 #include <iostream>
+#include "memory/memorymanager.h"
+#include "log/logger.h"
+#include "types/sizedtypes.h"
+#include "utility/singleton.h"
 namespace kurobako::engine
 {
     void InitEngine()
     {
-		sandcastle::graphics::simpletriangle triangle;
-		triangle.run();
-
-
-		int i = 0;
-		i = 3;
-		++i;
-
-		int*p = nullptr;
-#if defined(KUROBAKODEBUG)
-		*p = 3;
+        constexpr uint64 allocation_size = static_cast<uint64>(2) << 30;
+	    kurobako::memory::MemoryManager::InitializeMemoryManager(allocation_size);
+	    CREATE_SINGLETON_SCOPED(kurobako::log, Logger);
+	
+#if defined(KBK_DEBUG)
```

Comment:
I was using KUROBAKO but it became too long.

Lets fix it as

KBK_MACRO_DEFINE
kbkMacroFunction